packages/ez-react/src/docs/3_yourfirstchart.stories.mdx

```diff
@@ -0,0 +1,117 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Get Started/Your First Chart" parameters={{ options: { isToolshown: false }, id: 3 }} />
+
+# Your First Chart
+
+Now that you installed **EazyChart**, you will have the ability to use it and add a chart in your **React** or **Vue** web project. 
+
+EazyChart provides a set of ready-to-use charts that you can fully customize. In the example below, you will be adding a "Bar Chart" to a React project. First, you need to import the **BarChart** component from the library.
+
+```
+# Code for importing the bar chart component
+Import { BarChart } from 'eazychart-react'
+Import 'eazychart-css'
+```
+
+The above code will import the Bar chart to where you want it to show. 
```

Comment:
```suggestion
EazyChart provides a set of ready-to-use charts that you can fully customize. In the example below, you will be adding a "Bar Chart" to a React project. First, you need to import the **BarChart** component from the library.

```js
// Code for importing the bar chart component
Import { BarChart } from 'eazychart-react'
Import 'eazychart-css'
```

The above code will import the Bar chart to where you want it to show. 
```